src/js/components/Select/stories/Clear.js

```diff
@@ -4,29 +4,45 @@ import { Box, Grommet, Select, grommet } from 'grommet';
 
 const options = ['one', 'two', 'three'];
 
-const ClearTop = () => {
-  const [value, setValue] = useState();
+export const Top = () => {
+  const [value, setValue] = useState(options[0]);
   return (
-    <Box fill align="center" justify="start" pad="large">
-      <Select
-        placeholder="Clear Options"
-        multiple
-        value={value}
-        options={options}
-        onChange={({ value: nextValue }) => setValue(nextValue)}
-        clear
-      />
-    </Box>
+    <Grommet theme={grommet}>
```

Comment:
I would add a custom theme to one of these stories so others are able to see the behavior of being able to style clear option

Reply:
Alright, will do

Reply:
Will add a separate story for one styled with a custom theme